warp/Network/Wai/Handler/Warp/HTTP2.hs

```diff
@@ -106,10 +106,6 @@ http2server label settings ii transport addr app h2req0 aux0 response = do
             logResponse req st msiz
             mapM_ (logPushPromise req) pps
         Left e
-          -- killed by the local worker manager
-          | Just E.ThreadKilled  <- E.fromException e -> return ()
```

Comment:
I think this is very old code.
Currenlty, `killThread` is not used anywhere.

Reply:
But people can implement their own forking behaviour with `settingsFork`. So it COULD happen that a `ThreadKilled` is thrown to this thread.

Though now that I think about it, this is actually better, right? Since if someone would want to kill an HTTP thread that's handling a request, this would just ignore the signal. So I now also believe this is the right decision.